internal/db/collection_index.go

```diff
@@ -422,34 +422,14 @@ func (c *collection) dropIndex(ctx context.Context, indexName string) error {
 		}
 	}
 	key := keys.NewCollectionIndexKey(immutable.Some(c.Description().RootID), indexName)
-	err = txn.Systemstore().Delete(ctx, key.ToDS())
+	err = txn.Systemstore().Delete(ctx, key.Bytes())
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func (c *collection) dropAllIndexes(ctx context.Context) error {
```

Comment:
Question: This is no longer needed? What replaces this?

Reply:
It is no longer needed (private function not called anywhere).